app/react/Metadata/helpers/formater.js

```diff
@@ -137,12 +136,10 @@ export default {
   },
 
   preview(property, _value, _thesauris, { doc }) {
-    const reloadHack = doc.file && doc.file.filename ? doc.file.filename : '';
-    return this.multimedia(
-      property,
-      [{ value: `/api/attachment/${doc._id}.jpg${reloadHack ? `?r=${reloadHack}` : ''}` }],
-      'image'
-    );
+    // the fuck ?
+    // const reloadHack = doc.file && doc.file.filename ? doc.file.filename : '';
+    const defaultDoc = doc.defaultDoc || {};
+    return this.multimedia(property, [{ value: `/api/files/${defaultDoc._id}.jpg` }], 'image');
```

Comment:
use reloadhack ?? remove coment

Reply:
reloadHack should not be necessary anymore, changing file changes also the thumbnail name.